.changeset/lovely-jeans-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+#nops : Enable configurable client error regexes for error classification
```

Comment:
Oh this should reference the new toml config field so they can find the feature.